modules.json

```diff
@@ -144,8 +144,7 @@
                     },
                     "gatk4/cnnscorevariants": {
```

Comment:
Should there also be an update to the sha/version of the multiqc-module?

Reply:
done, I actually removed the patch I was using before